nb_iterations: 3 # Number of iterations of the path to be executed
step_by_step: false # If True, the robot will ask for enter to move to the next step
max_path_trial: 10 # Maximum number of trials to generate a cartesian path